datafusion_iceberg/src/materialized_view/delta_queries/delta_node.rs

```diff
@@ -37,15 +37,6 @@ impl UserDefinedLogicalNodeCore for PosDeltaNode {
         write!(f, "PosDelta")
     }
 
-    fn from_template(&self, exprs: &[Expr], inputs: &[LogicalPlan]) -> Self {
```

Comment:
This function does not exist anymore on `UserDefinedLogicalNodeCore`. I couldn't find a replacement API, so I just deleted it. I'm not sure if that is correct, though